Library/Formula/wordgrinder.rb

```diff
@@ -0,0 +1,15 @@
+class Wordgrinder < Formula
+  homepage "http://wordgrinder.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/wordgrinder/wordgrinder/wordgrinder-0.5.2.tar.bz2"
+  sha1 "4bcc600fbd110d9d87961feab15045b9b518b2db"
+
+  depends_on "lua"
+  depends_on "ncurses" => :build
+
+  def install
+    system "make"
```

Comment:
There is a `make install` but it uses GNU install, which uses a capital letter for at least one flag, and fails on BSD install on OSX. That's why I did the other commands below since it should do the same thing.